### PR TITLE
Fix Rubocop Scope Problem

### PR DESCRIPTION
**Pull request by nnhimes**
~~To make rubocop only run on changed files in a PR~~
Run certain rubocop cops on app/ files, with ability to add more cops in the future

### CODE REVIEW

**Comment by nnhimes** [comment]
Check https://docs.rubocop.org/rubocop/cops.html for any more cops that should be added. We can either make the fixes ~100-200 in this PR or make a new one

File: app/models/quiz_response_map.rb [hunk 1]
```diff
@@ -53,5 +53,6 @@ def quiz_score
 
     # convert the obtained percentage to float and round it to 1st precision
     weighted_quiz_score = calculated_score[0].graded_percent.to_f.round(1)
+    weighted_quiz_score
```
Comment (johnbumgardner) [inline app/models/quiz_response_map.rb:56]:
```suggestion

    weighted_quiz_score = calculated_score[0].graded_percent.to_f.round(1)
    weighted_quiz_score
```
```suggestion
    calculated_score[0].graded_percent.to_f.round(1)
```

Comment (johnbumgardner) [reply app/models/quiz_response_map.rb:56]: stop using the weighted quiz score

File: app/models/review_response_map.rb [hunk 2]
```diff
@@ -9,6 +9,7 @@ class ReviewResponseMap < ResponseMap
   def after_initialize
     # If an assignment supports team reviews, it is marked in each mapping
     reviewer_is_team = assignment.reviewer_is_team
+    reviewer_is_team
```
Comment (johnbumgardner) [inline app/models/review_response_map.rb:12]:
```suggestion
    reviewer_is_team
```
```suggestion

```

File: app/models/review_response_map.rb [hunk 3]
```diff
@@ -9,6 +9,7 @@ class ReviewResponseMap < ResponseMap
   def after_initialize
     # If an assignment supports team reviews, it is marked in each mapping
     reviewer_is_team = assignment.reviewer_is_team
```
Comment (johnbumgardner) [inline app/models/review_response_map.rb:11]:
```suggestion
    reviewer_is_team = assignment.reviewer_is_team
```
```suggestion
    assignment.reviewer_is_team
```

**Review by johnbumgardner — CHANGES_REQUESTED** [review]

**Pull request merged**